facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -35,6 +35,15 @@ <h3 class="headline">
         @if(card.header.quoted) {
             @card.bylineText.map { byline => <h4 class="byline">@byline</h4> }
         }
+        @card.starRating.map { numberOfStars =>
+            <div class="review-stars">
+                @for(i <- 0 to 4) {
+                    @defining(if(i < numberOfStars) "positive" else "negative") { positiveOrNegative =>
```

Comment:
Went for `golden` and `grey` for consistency with existing review stars: https://github.com/guardian/frontend/blob/master/common/app/views/fragments/items/elements/starRating.scala.html#L5